src/docs/blueprint_documentation.md

```diff
@@ -0,0 +1,182 @@
+---
+title: Packaging Theia as a Desktop Product
+---
+
+# Packaging Theia as a Desktop Product
+
+Eclipse Theia Blueprint is an example product used as a reference on how to build desktop IDE-like products based on the Eclipse Theia framework.
+Theia Blueprint assembles a selected subset of existing Theia features and extensions.
+We provide installers for Theia Blueprint to be downloaded (see links below).
+In the respective git repository you can also find the source code for the product and its installers.
+This documentation will use these sources as a template. We will explain
+how to customize this template so you can build your own custom Theia-based product including installers and packaging for installing the desktop-based version of your custom product on all major operating systems.
+
+## Building a product and installers
+
+The Theia Blueprint build uses [electron-builder](https://www.electron.build/) to package the product as a desktop application.
+
+The product can be built and packaged with [yarn](https://yarnpkg.com/).
+Note that you usually can only package the product for the operating system you execute the build on.
+For more information see the [electron-builder documentation on multi platform builds](https://www.electron.build/multi-platform-build).
+
+The following commands may be run from the root directory of the repository.
+
+To install dependencies and build the application simply execute `yarn`.
+
+You can also directly run the unpackaged application, e.g. to test it during development with `yarn start`.
+
+With `yarn package` you package the application into an executable file for your current operating system.
+The packaged application will be located in `electron-app/dist`.
+The folder `electron-app/dist/<OS>-unpackaged` will contain the files that are bundled into the final packaged executable.
+For Linux, this is an executable `.AppImage`, for Windows a `.exe` installer, and a `.pkg` installer for MacOS.
+
+You can also only create the unpackaged content by running `yarn package:preview`.
+This is useful to see the bundled files and saves time compared to a full package.
+To publish the current version of the product the command `yarn deploy` can be used.
+For more information on publishing also see section "Configure publish and update".
```

Comment:
Will this be a link?

Reply:
As mentioned here https://github.com/eclipse-theia/theia-website/pull/133#discussion_r567837459 translating markdown links to html links does not work with the current set of plugins generating the HTML-Page. At least as far as I could see. So unfortunatelly this won't be a link.